client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -296,20 +272,290 @@ class EmailProvidersComparison extends React.Component {
 						  } )
 						: null
 				}
-				buttonLabel={ buttonLabel }
 				onButtonClick={ this.goToAddGSuite }
 				className={ classNames( className, 'gsuite' ) }
+				{ ...gsuiteProps }
+			/>
+		);
+	}
+
+	getGoogleDetailProps() {
+		const { translate } = this.props;
+
+		if ( config.isEnabled( 'google-workspace-migration' ) ) {
+			return {
+				title: getGoogleMailServiceFamily(),
+				description: translate(
+					'The best way to create, communicate, and collaborate. An integrated workspace that is simple and easy to use.'
+				),
+				productLogo: googleWorkspaceIcon,
+				buttonLabel: translate( 'Add %(googleMailService)s', {
+					args: {
+						googleMailService: getGoogleMailServiceFamily(),
+					},
+					comment: '%(googleMailService)s can be either "G Suite" or "Google Workspace"',
+				} ),
+			};
+		}
+
+		return {
+			title: translate( 'G Suite by Google' ),
+			description: translate(
+				"We've partnered with Google to offer you email, storage, docs, calendars, and more."
+			),
+			productLogo: gSuiteLogo,
+			buttonLabel: translate( 'Add G Suite' ),
+		};
+	}
+
+	getForwardingFeatures() {
+		const { translate } = this.props;
+
+		return [ translate( 'No billing' ), translate( 'Receive emails sent to your custom domain' ) ];
+	}
+
+	getGoogleFeatures() {
+		const { translate } = this.props;
+
+		return [
+			translate( 'Annual billing' ),
+			translate( 'Send and receive from your custom domain' ),
+			translate( '30GB storage' ),
+			translate( 'Email, calendars, and contacts' ),
+			translate( 'Video calls, docs, spreadsheets, and more' ),
+			translate( 'Work from anywhere on any device – even offline' ),
+		];
+	}
+
+	getTitanFeatures() {
+		const { translate } = this.props;
+
+		return [
+			translate( 'Monthly billing' ),
+			translate( 'Send and receive from your custom domain' ),
+			translate( '30GB storage' ),
+			translate( 'Email, calendars, and contacts' ),
+			translate( 'One-click import of existing emails and contacts' ),
+			translate( 'Apps for Android and iOS' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **4** times:
`translate( 'Jetpack for Android and iOS' )` *ES Score: 7*